make quick-release-images fails when building conformance image

```
+++ [0704 16:31:48] Building conformance image for arch: amd64
cp: cannot stat '/home/dims/go/src/k8s.io/kubernetes/cluster/images/conformance/../../../_output/dockerized/bin/linux/amd64/go-runner': No such file or directory
make[1]: *** [build] Error 1
```

--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -109,6 +109,7 @@ kube::golang::conformance_image_targets() {
   local targets=(
     vendor/github.com/onsi/ginkgo/ginkgo
     test/e2e/e2e.test
+    cluster/images/conformance/go-runner
     cmd/kubectl
   )
   echo "${targets[@]}"

